Applications/Quickstarts.Servers/SampleNodeManager/SampleNodeManager.cs

```diff
@@ -2998,6 +2970,59 @@ IList<IMonitoredItem> monitoredItems
             // does nothing.
         }
 
+        /// <summary>
+        /// ReadInitial values.  
+        /// </summary>
+        /// <param name="systemContext">The context.</param>
+        /// <param name="monitoredItem">The datachange monitored item for which ReadInitialValue is initiated.</param>
+        /// <param name="errorCode">Any error.</param>
+        /// <param name="processedItem">The bool stating if already processed.</param>
+        /// <param name="transferredItems">The transferred monitored items.</param>
+        /// <returns></returns>
+        protected override Tuple<ServiceResult, bool, IList<IMonitoredItem>> DoReadInitialValue(
+           ServerSystemContext systemContext,
+           IMonitoredItem monitoredItem,
+           ServiceResult errorCode,
+           bool processedItem,
+           IList<IMonitoredItem> transferredItems)
+        {
+            Tuple< ServiceResult, bool, IList<IMonitoredItem>> result =
+                new Tuple<ServiceResult, bool, IList<IMonitoredItem>>(StatusCodes.BadMonitoredItemIdInvalid, false, transferredItems);
+
+            if (!monitoredItem.IsReadyToPublish)
+            {
+                bool rProcessedItem = false;
+                ServiceResult rError = StatusCodes.BadMonitoredItemIdInvalid;
```

Comment:
what does r stand for? its a new naming convention i the codebase.

Reply:
Will be corrected